roles/telemetry_verify_metrics/tasks/main.yml

```diff
@@ -0,0 +1,30 @@
+---
+# tasks file for telemetry_verify_metrics
```

Comment:
If you follow the suggestion below to add the contents of this file to the common role, you can replace this task with:

```
- name: Verify OpenStack is deployed correctly
  ansible.builtin.include_role:
     name: common
     tasks_from: check_openstack_services_rhoso.yml
  vars:
        - [<CR>, <resource name>]
        - [<CR>, <resource_name>]
```

The data structure may be a list of dicts as well so it's easier to grok, but I think the list-of-lists might be easier to input.